cfpb/consumerfinance.gov
# Regs3k: update dependencies to fix security warning

The `browserslist` and `lodash` dependencies in regs3k app have security warnings 
https://github.com/cfpb/consumerfinance.gov/security/dependabot

This PR refreshes the yarn lock and npm cache to pull in the latest. It also updates workbox while we're in here and adds a url for the workbox script.

## Changes

- Regs3k: refresh npm cache.
- Update to "workbox-build": "6.1.5" from "6.0.2"
- Adds url for workbox script to urls.py

## How to test this PR

1. Delete `cfgov/regulations3k/jinja2/regulations3k/regulations3k-service-worker.js` and any `workbox-` scripts and sourcemap files.
1. Pull branch and run `gulp clean && gulp build`.
1. Visit a regs3k page like http://localhost:8000/rules-policy/regulations/1002/3/ and see that there are no errors in the console and it works as expected.
